### DISCUSSION TITLE
0.10.0 published!

### DISCUSSION THREAD

**Discussion by PaulTR** [tensorflow/flutter-tflite#57] [Announcements]
Quite a few changes, especially bringing tflite to a current version and removing the need to download .so files individually. Most changes should be in the change log.

As always, issues and additional pull requests are more than welcome :)

Thanks!

**Comment by sruizhuenei** [comment]
Hi Paul, could you tell me what is the helper library.
I'm a little stuck with an old project that use this library and a tflite_flutter_helper, but this one is deprecated :(

Best!

Sergio Ruiz.

**Reply by PaulTR** [reply]
Helper was basically a wrapper for a few specific tasks in the TFLite plugin (pretty much like the TensorFlow Lite Task Library). When we moved this repo to an official repo so we can have admins assigned to it, we decided to leave the helper behind because it was already broken (along with this plugin being broken), and the original idea was to point people to the MediaPipe library for those out-of-the-box tasks. The MP library is still being developed, albeit with a lot less resources behind it now than I'd like, but we made sure to include examples that show how to do anything that the helper also did in this plugin.

**Reply by sruizhuenei** [reply]
OK, so I need to install this plugin and try to remove the use of the helper librery...see what braeks and try to solve it...
Thanks Paul.

I'll be in touch :)

Best!

Sergio Ruiz

**Reply by sruizhuenei** [reply]
hi @PaulTR I was able to build the app and running :)
Now I have to change my old model with new model trained with Yolo V8, did you now if this package support this type of model?

Thanks Paul

Regards!
